### PR TITLE
Add paging examples

### PR DESCRIPTION
**Pull request by deepd**
- Add an example of paging over all devices' sub-interfaces' neighbors
and print them
- Add an example of paging over all devices' MAC table entries and
print them

### CODE REVIEW

File: examples/show_mac_table.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+          }
+        }
+      }
+    }
+  }
+}
+'''
+
+# Query to get a device's default network-instance's next page of mac entries.
+subQuery = Template('''
```
Comment (AndreasVoellmy) [inline examples/show_mac_table.py]: Use GraphQL variables for this. See https://graphql.org/learn/queries/#variables

The FwdApi object has a method api.queryWithVars() that you can use that takes a final parameter that will be passed as arguments.

That way you don't have to do string interpolation here.

Also, that will allow you to use a single query for both the initial and the subsequent queries.

Comment (deepd) [reply examples/show_mac_table.py]: Done.

File: examples/show_arp_entries.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                print_neighbor(item)
+            pageInfo = s['ipv4']['neighborsPage']['pageInfo']
+            cursor = pageInfo['endCursor']
+            # Iterate over all the pages
+            while pageInfo['hasNextPage']:
+                subDataset =\
+                    api.queryWithVars(args.snapshotId, query,
+                                      {'deviceName': deviceName,
+                                       'interfaceName': interfaceName,
+                                       'subInterfaceName': subInterfaceName,
```
Comment (deepd) [inline examples/show_arp_entries.py]: `subinterfaceName` and not `subInterfaceName`

Comment (deepd) [reply examples/show_arp_entries.py]: Done

File: examples/show_sub_interfaces_ipv4_neighbors.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+initialQuery = '''
+{
+  devices {
+    name
+    interfaces {
+      name
+      subinterfaces {
+        name
+        ipv4 {
+          neighborsPage {
```
Comment (nikhilh) [inline examples/show_sub_interfaces_ipv4_neighbors.py]: neighborsPage can be found under other subtrees of the interfaces model as well, including `routedVlan`, `bridge`, and `tunnel`. Can you update the code to extract ARP entries from those as well?

File: examples/show_sub_interfaces_ipv4_neighbors.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
```
Comment (nikhilh) [inline examples/show_sub_interfaces_ipv4_neighbors.py]: It'd be easier to understand if this file was named `show_arp_table.py`.

Comment (deepd) [reply examples/show_sub_interfaces_ipv4_neighbors.py]: Done in latest.

**Pull request closed without merging**